### PR TITLE
Rename type Keystone to KeystoneSystem

### PR DESCRIPTION
**Pull request by timleslie**
Having a type called `Keystone` which is different to the core class `Keystone` is confusing, particularly when a field of the type is an instance of the `Keystone` class. This renames the type `Keystone` to `KeystoneSystem` to avoid this confusion.

### CODE REVIEW

**Review by JedWatson — CHANGES_REQUESTED** [review]
👍 LGTM, but only the `types` package should be major, right? `keystone` and `admin-ui` should be patch because they have no API change

**Comment by timleslie** [comment]
> only the `types` package should be major, right? `keystone` and `admin-ui` should be patch because they have no API change

Yeah, I was in two minds about this. In the past we've had some weird issues where we `major` bumped one thing and `patch` bumped a package that used it. Going with a `major` bump is safer, but churnier. Probably not too much of an issue in these packages right now. Happy to change packages to `patch` and see what happens.

File: .changeset/cyan-terms-unite.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+'@keystone-next/admin-ui': major
+'@keystone-next/keystone': major
```
Comment (timleslie) [inline .changeset/cyan-terms-unite.md:3]:
```suggestion
'@keystone-next/admin-ui': patch
'@keystone-next/keystone': patch
```

**Review by JedWatson — APPROVED** [review]
👍

**Pull request merged**